src/repr/src/adt/mz_acl_item.rs

```diff
@@ -0,0 +1,347 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+//! A Materialize specific access control list abstract data type.
+
+use crate::role_id::RoleId;
+use anyhow::{anyhow, Error};
+use bitflags::bitflags;
+use columnation::{CloneRegion, Columnation};
+use mz_proto::{RustType, TryFromProtoError};
+use regex::Regex;
+use serde::{Deserialize, Serialize};
+use std::fmt;
+use std::mem::size_of;
+use std::ops::BitOrAssign;
+use std::str::FromStr;
+
+include!(concat!(env!("OUT_DIR"), "/mz_repr.adt.mz_acl_item.rs"));
+
+const INSERT_CHAR: char = 'a';
+const SELECT_CHAR: char = 'r';
+const UPDATE_CHAR: char = 'w';
+const DELETE_CHAR: char = 'd';
+const USAGE_CHAR: char = 'U';
+const CREATE_CHAR: char = 'C';
+
+bitflags! {
+    /// A bit flag representing all the privileges that can be granted to a role.
+    ///
+    /// Modeled after:
+    /// https://github.com/postgres/postgres/blob/7f5b19817eaf38e70ad1153db4e644ee9456853e/src/include/nodes/parsenodes.h#L74-L101
+    #[derive(Serialize, Deserialize)]
+    pub struct AclMode: u64 {
+        const INSERT = 1 << 0;
+        const SELECT = 1 << 1 ;
```

Comment:
```suggestion
        const SELECT = 1 << 1;
```

Reply:
Fixed (surprised the linter didn't catch this).